tests/test-rest-controller.php

```diff
@@ -15,6 +15,14 @@ public function setUp() {
 				'somestring'      => array(
 					'type'        => 'string',
 				),
+				'somearray'       => array(
+					'type'        => 'array',
+					'allow_csv'   => false,
+				),
+				'somecsv'         => array(
+					'type'        => 'array',
+					'allow_csv'   => true,
+				),
```

Comment:
We need to test against the actual JSON Schema here.  So we need to have items added.  In a PR I did that I ultimately did not like I had schema like this.

`someIdList => array(
    'type' => 'array',
    'items' => array(
        'type' => 'integer'
    ),
   'minItems' => 1,
   'uniqueItems' => true,
),`